sdk/keyvault/Azure.Security.Keyvault.Secrets/src/Azure.Security.Keyvault.Secrets.csproj

```diff
@@ -5,7 +5,7 @@
     <Description>This is the Microsoft Azure Key Vault Secrets client library</Description>
     <AssemblyTitle>Microsoft Azure.Security.KeyVault.Secrets client library</AssemblyTitle>
     <Version>1.0.0-preview.1</Version>
-    <PackageTags>Microsoft Azure Application Configuration</PackageTags>
+    <PackageTags>Microsoft Azure Key Vault Secrets</PackageTags>
     <PackageReleaseNotes>
```

Comment:
The TreatWarningsAsErrors, RootNamespace and AssembyName properties are redundant and can be removed.  I'd also check to see if the NoWarn and AllowUnsafeBlocks properties are necessary and if not remove them as well.

Reply:
After PR https://github.com/Azure/azure-sdk-for-net/pull/6123/files we want to have  `TreatWarningsAsErrors` enabled.
As for `NoWarn` and `AllowUnsafeBlocks ` removing it causes noises related to code in `Microsoft.Bcl.Json.Sources`

Reply:
I thought I had enabled TreatWarningsAsErrors globally but I guess I didn't yet. I only enabled it in the build scripts https://github.com/Azure/azure-sdk-for-net/pull/5351. I'm fine with it staying for now but I will want to enable it globally at some point.

Reply:
sounds good. Happy to remove it later